build/build.mjs

```diff
@@ -0,0 +1,37 @@
+import { DECORATORS } from './constants.mjs';
```

Comment:
I wonder if it makes sense to push this stuff into a tool? Maybe write grabthar? lol